src/components/Atoms/Picture/Picture.js

```diff
@@ -58,7 +58,7 @@ const Picture = ({ images, alt, theme, width, height, objectFit }) => {
 };
 
 Picture.propTypes = {
-  images: PropTypes.object.isRequired,
+  images: PropTypes.oneOfType([PropTypes.string, PropTypes.object]).isRequired,
```

Comment:
I thought `object` wasn't allowed in the ESlint rules we're using?

Reply:
It's not .. there is a ```eslint-disable``` line at the top :) .. let me fix that with shape

Reply:
ok, it was ignoring due `theme` type .. I've added a shape type for `theme` as well . I think it works.